.github/workflows/build.yml

```diff
@@ -174,14 +224,17 @@ jobs:
         chmod +x Linux-binaries/*
         zip -j release/fossa_${{ steps.get-version.outputs.VERSION }}_linux_amd64.zip Linux-binaries/fossa
         zip -j release/pathfinder_${{ steps.get-version.outputs.VERSION }}_linux_amd64.zip Linux-binaries/pathfinder
+        zip -j release/compat-attribution_${{ steps.get-version.outputs.VERSION }}_linux_amd64.zip Linux-binaries/compat-attribution
 
         chmod +x macOS-binaries/*
         zip -j release/fossa_${{ steps.get-version.outputs.VERSION }}_darwin_amd64.zip macOS-binaries/fossa
         zip -j release/pathfinder_${{ steps.get-version.outputs.VERSION }}_darwin_amd64.zip macOS-binaries/pathfinder
+        zip -j release/compat-attribution_${{ steps.get-version.outputs.VERSION }}_linux_amd64.zip macOS-binaries/compat-attribution
 
         chmod +x Windows-binaries/*
         zip -j release/fossa_${{ steps.get-version.outputs.VERSION }}_windows_amd64.zip Windows-binaries/fossa.exe
         zip -j release/pathfinder_${{ steps.get-version.outputs.VERSION }}_windows_amd64.zip Windows-binaries/pathfinder.exe
+        zip -j release/compat-attribution_${{ steps.get-version.outputs.VERSION }}_linux_amd64.zip Windows-binaries/compat-attribution.exe
```

Comment:
I have not tested this bit - but should this end with `_windows_amd64.zip Windows-binaries/compat-attribution.exe` instead?